src/sections/Header.html

```diff
@@ -0,0 +1,29 @@
+<section
+  class="flex flex-row md:justify-between md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
+>
+  <div class="flex flex-col text-[1rem] md:text-[1.6rem] lg:text-[1.5rem]">
+    <p>DE</p>
+    <p>EN</p>
+  </div>
+  <div class="pl-[25%] lg:pl-[20%] xl:pl-[40%] md:mr-24">
+    <h1 class="font-semibold text-[1.5rem] lg:text-[3rem] md:text-[3rem]">
+      Baumeister
+    </h1>
+  </div>
+  <div class="text-[1.5rem] pl-[33%] lg:pl-[25%] lg:flex xl:flex lg:flex-col xl:flex-col hidden">
```

Comment:
I was wondering why you used arbitrary values for text sizes instead of the usual built-in tailwind text-sizes. I think it would be better to decrease the amount of arbitrary values you use.

Reply:
Thanks we did change it